### PR TITLE
new-files

### PR DESCRIPTION
**Pull request by SaadMahi**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+:root {
+  --main-radius: 5px;
+  --main-padding: 5px;
+}
+
```
Comment (ManSOorcode) [inline style.css:5]: add this in your css :
```js
*{
  margin:0;
  padding:0;
  box-sizing:border-box;
};
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  grid-gap: 0.2rem;
+
+  font-family: "inter", sans-serif;
+  font-weight: 800;
+  text-transform: uppercase;
+  font-size: 0.75em;
+
+  text-align: center;
+
```
Comment (ManSOorcode) [inline style.css:34]: add this in your css:
```js
padding: var(--main-padding);
```
so this will remove the scroll bar which come on your screen

**Review by ManSOorcode — CHANGES_REQUESTED** [review]
done this above change and it will remove the scroll bar from your grid project

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (murtazshaikh) [inline index.html]: set title according to project name. for example: <title>Responsive Grid Layout</title>

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <link
+      href="https://fonts.googleapis.com/css?family=Inter:400,800,900&display=swap"
+      rel="stylesheet"
+    />
+    <!--Css Style Sheet-->
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav>NavBar</nav>
```
Comment (murtazshaikh) [inline index.html:18]: Read more about semantic tags: https://www.semrush.com/blog/semantic-html5-guide/#html-semantic-tags-for-structure

**Review by murtazshaikh — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (SaadMahi) [reply style.css:5]: i would like to know in detail what is this used for?

File: style.css (hunk 2 above)
Comment (SaadMahi) [reply style.css:34]: padding: var(--main-padding); added

File: index.html (hunk 4 above)
Comment (SaadMahi) [reply index.html:18]: great! have saved the link will definitely go through it.

**Review by SaadMahi — COMMENTED** [review]
CSS resolution pending in which we use universal tag *.

File: style.css (hunk 1 above)
Comment (SaadMahi) [reply style.css:5]: added

**Review by SaadMahi — COMMENTED** [review]
new-changes made to style.css. Those are using universal selector below root:

**Review by murtazshaikh — APPROVED** [review]
Great @SaadMahi

**Comment by SaadMahi** [comment]
> Great @SaadMahi

looks like page isn't mobile responsive any more due to the above changes

**Comment by SaadMahi** [comment]
> Great @SaadMahi

Kindly view page deployment, the page isn't displaying as desired, i think it's due to the use of universal selector.

**Review by ManSOorcode — APPROVED** [review]
LGTM

**Pull request merged**